Doc/library/sqlite3.rst

```diff
@@ -908,7 +911,7 @@ Cursor Objects
 
    .. method:: executescript(sql_script, /)
 
-      Execute multiple SQL statements at once.
+      Execute the multiple SQL statements in *sql_script*.
```

Comment:
```suggestion
      Execute the SQL statements in *sql_script*.
```

"Multiple" sounds somewhat strange and confusing to me here, and given "statements" is already plural it seems it can just be elided, no?